pkg/cmd/pr/create/create.go

```diff
@@ -134,8 +134,8 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 	fl.StringVarP(&opts.HeadBranch, "head", "H", "", "The `branch` that contains commits for your pull request (default: current branch)")
 	fl.BoolVarP(&opts.WebMode, "web", "w", false, "Open the web browser to create a pull request")
 	fl.BoolVarP(&opts.Autofill, "fill", "f", false, "Do not prompt for title/body and just use commit info")
-	fl.StringSliceVarP(&opts.Reviewers, "reviewer", "r", nil, "Request reviews from people by their `login`")
-	fl.StringSliceVarP(&opts.Assignees, "assignee", "a", nil, "Assign people by their `login`")
+	fl.StringSliceVarP(&opts.Reviewers, "reviewer", "r", nil, "Request reviews from people by their `login`, or teams org/teamname")
```

Comment:
Good catch! I didn't even notice that our docs make it sound like only users can get requested for review.

Could the docs be something like this:
```suggestion
	fl.StringSliceVarP(&opts.Reviewers, "reviewer", "r", nil, "Request reviews from people or teams by their `handle`")
```
The flag synopsis will then be rendered like so: `-r, --reviewer handle`, and I feel like the word "handle" encompasses both user login handle and the team slug. Does that sound reasonable?

Reply:
Yes, absolutely! "login" threw me off because teams don't have logins. Handle is great!